File: workspace-servidor/flotix2021/src/main/java/com/flotix/response/bean/ServerResponseImagenVehiculo.java
package com.flotix.response.bean;

import com.flotix.firebase.model.ImagenVehiculo;

public class ServerResponseImagenVehiculo {

	private String idImagenVehiculo = null;

	private ImagenVehiculo imagenVehiculo = null;

	private ErrorBean error = new ErrorBean();

	public ServerResponseImagenVehiculo() {
		super();
	}

	public String getIdImagenVehiculo() {
		return idImagenVehiculo;
	}

	public void setIdImagenVehiculo(String idImagenVehiculo) {
		this.idImagenVehiculo = idImagenVehiculo;
	}

	public ImagenVehiculo getImagenVehiculo() {
		return imagenVehiculo;
	}

	public void setImagenVehiculo(ImagenVehiculo imagenVehiculo) {
		this.imagenVehiculo = imagenVehiculo;
	}

	public ErrorBean getError() {
		return error;
	}

	public void setError(ErrorBean error) {
		this.error = error;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelResponse/ServerResponseImagenVehiculo.cs
using Flotix2021.Model;

namespace Flotix2021.ModelResponse
{
    public class ServerResponseImagenVehiculo
    {
        private string _idImagenVehiculo;

        public string idImagenVehiculo
        {
            get { return _idImagenVehiculo; }
            set { _idImagenVehiculo = value; }
        }

        private ImagenVehiculo _imagenVehiculo;

        public ImagenVehiculo imagenVehiculo
        {
            get { return _imagenVehiculo; }
            set { _imagenVehiculo = value; }
        }

        private ErrorBean _error;

        public ErrorBean error
        {
            get { return _error; }
            set { _error = value; }
        }
    }
}
